packages/exhibitor-migration-script/README.md

```diff
@@ -0,0 +1,48 @@
+exhibitor-migration-script
+==========================
+
+When a user is updating from DC/OS 1.7 or 1.8 we need to update several values in exhibitor to use the new filesystem
+paths for zookeeper state on disk.
+
+This package includes two scripts to help with managing the update.
+
+
+## `dcos-exhibitor-migrate-status`
+
+Will query exhibitor (at `localhost`) and check if the migration has been completed, in progress, or has not yet been
+performed.
+
+| exit code | State |
+| --------- | ----- |
+| 0         | Exhibitor has new expected values and no rollout is in progress |
+| 1         | IO Error when trying to connect to exhibitor (expected if run on an agent) |
+| 2         | Error when reading from exhibitor (Any response other than 200) |
+| 4         | Rolling update in progress |
+| 8         | At least one config value does not have the expected post migration value |
+
+
+This script can be run on any of the DC/OS Master nodes by running
+
+```
+dcos-shell /opt/mesosphere/bin/dcos-exhibitor-migrate-status
```

Comment:
Do we need the absolute path if we're wrapping it with `dcos-shell`? I thought that would put `/opt/mesosphere/bin` in your `PATH`.